tests/evm_benchmark_test.go

```diff
@@ -0,0 +1,173 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/require"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/state"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+const (
+	benchmarksDir = "evm-benchmarks/benchmarks"
+	chainID       = 10
+)
+
+func BenchmarkEVM(b *testing.B) {
+	folders, err := listFolders([]string{benchmarksDir})
+	require.NoError(b, err)
+
+	for _, folder := range folders {
+		files, err := listFiles(folder, ".json")
+		require.NoError(b, err)
+
+		for _, file := range files {
+			name := getTestName(file)
+
+			b.Run(name, func(b *testing.B) {
+				data, err := os.ReadFile(file)
+				require.NoError(b, err)
+
+				var testCases map[string]testCase
+				if err = json.Unmarshal(data, &testCases); err != nil {
+					b.Fatalf("failed to unmarshal %s: %v", file, err)
+				}
+
+				for _, tc := range testCases {
+					for fork, postState := range tc.Post {
+						forks, exists := Forks[fork]
+						if !exists {
+							b.Logf("%s fork is not supported, skipping test case.", fork)
+							continue
+						}
+
+						fc := &forkConfig{name: fork, forks: forks}
+
+						for idx, postStateEntry := range postState {
+							err := runBenchmarkTest(b, tc, fc, postStateEntry)
+							require.NoError(b, err, fmt.Sprintf("test %s (case#%d) execution failed", getTestName(file), idx))
```

Comment:
You already gotten the name of the test:
```suggestion
							require.NoError(b, err, fmt.Sprintf("test %s (case#%d) execution failed", name, idx))
```